File: Java/CodingBat/countHi.java
// http://codingbat.com/prob/p147448

public int countHi(String str) {
  int numberOfHellos = 0;
  for (int i = 0; i < str.length()-1; ++i) {
    if (str.substring(i, i+2).equals("hi")) {
      numberOfHellos++;
    }
  }
  return numberOfHellos;
}

File: Python/CodingBat/count_hi.py
# http://codingbat.com/prob/p167246

def count_hi(str):
  hi_count = 0
  
  for i in range(len(str)-1):
    if (str[i:i+2] == "hi"):
      hi_count += 1
  
  return hi_count
